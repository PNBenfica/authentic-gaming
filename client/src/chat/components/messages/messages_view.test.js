import renderer from 'react-test-renderer';
import MessagesView from './messages_view';

const messages = [
  {
    userName: 'Paulo',
    messages: 'SLB',
    timestamp: 123456789
  }
]

it('renders correctly', () => {
  const tree = renderer
    .create(<MessagesView messages={messages} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders no messages warning', () => {
  const tree = renderer
    .create(<MessagesView messages={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});