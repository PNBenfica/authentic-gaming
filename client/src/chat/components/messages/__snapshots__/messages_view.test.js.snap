// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="box messages-container"
>
  <div
    className="message-container"
  >
    <p
      className="username"
    >
      Paulo
    </p>
    <p
      className="message"
    />
  </div>
</div>
`;

exports[`renders no messages warning 1`] = `
<div
  className="box messages-container"
>
  No messages yet
</div>
`;
